frontend/src/components/summary/group/ECEWESummary.vue

```diff
@@ -254,13 +241,41 @@ export default {
   computed: {
     ...mapState(useApplicationStore, ['formattedProgramYear']),
     ...mapState(useSummaryDeclarationStore, ['isLoadingComplete']),
+    ...mapState(useOrganizationStore, ['organizationProviderType']),
     ...mapState(useAppStore, ['fundingModelTypeList', 'getFundingUrl', 'getLanguageYearLabel']),
     languageYearLabel() {
       return this.getLanguageYearLabel;
     },
     programYearTypes() {
       return PROGRAM_YEAR_LANGUAGE_TYPES;
     },
+    showApplicableSector() {
+      return (
+        (this.ecewe?.belongsToUnion == 1 &&
+          this.ecewe?.optInECEWE == 1 &&
+          this.ecewe?.publicSector == 1 &&
+          this.languageYearLabel != this.programYearTypes.HISTORICAL) ||
+        (this.ecewe?.belongsToUnion == 1 &&
+          this.ecewe?.optInECEWE == 1 &&
+          this.languageYearLabel == this.programYearTypes.HISTORICAL)
+      );
+    },
+    showFundingModel() {
+      return (
+        this.ecewe?.optInECEWE == 1 && this.ecewe?.belongsToUnion == 1 && this.ecewe?.applicableSector == 100000000
```

Comment:
Do we want to have constants for the two applicableSector values so we know at a glance what they refer to? Can just be defined in the component if not used outside of it.

Reply:
yup, done